std/algorithm/iteration.d

```diff
@@ -2402,7 +2412,9 @@ version (none) // this example requires support for non-equivalence relations
         assert(!range.empty);    // Opposite of refInputRange test
     }
 
-    /* Issue 93532 - General behavior of non-forward input ranges.
+    /* https://issues.dlang.org/show_bug.cgi?id=93532
```

Comment:
We don't have 90k issues yet, right? ;)

Reply:
Real issue: 19532

Reply:
Thanks! I did see it, but couldn't find the real issue, and then forgot about it... Will fix.